frontend/app/openproject-app.js

```diff
@@ -30,7 +30,6 @@ var I18n = require('./vendor/i18n');
 
 // standard locales
 I18n.translations.en = require("locales/js-en.yml").en;
-I18n.translations.de = require("locales/js-de.yml").de;
```

Comment:
The reason for this difference is that the locales (except english) for the core are saved in OpenProject-Translations and the locales for the plugins are saved in the plugins themselves. Thus, the plugins load all locales and the core only loads english (OpenProject-Translation loads the rest).